app/views/admin_mailer/send_spam_alert.text.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for :message do %>
+
+The following accounts have a suspicious level of traffic:<
+ <%@spam_score.sort_by{|k,v| -1*v}.map do |pseud,score| %>
+ <% user = User.find(pseud) %>
+ User <%= user.login%> has a score of <%= score %>
```

Comment:
Should be `<%= user.login %>`. Also, I noticed there aren't any URLs in this email -- it might be a good idea to put the URL in parentheses, e.g. `<%= user.login %> (<%= root_url + user_works_path(user) %>) has a score of` and `<%= work.title %> (<%= work_url(work) %>)`